src/stores/TransactionStore.js

```diff
@@ -558,7 +558,7 @@ export default class TransactionStore {
           amount,
         });
 
-        await this.onTxExecuted(index, tx);
+        await this.onTxExecuted(index, tx, new Transaction(createBet));
```

Comment:
let's make a change to `mutations.js` where `createTransaction()` returns a `new Transaction()` instance.